aiohttp/client_reqrep.py

```diff
@@ -550,6 +556,16 @@ def _headers(self):
     def request_info(self):
         return self._request_info
 
+    @reify
+    def content_disposition(self):
+        raw = self._headers.get(hdrs.CONTENT_DISPOSITION)
+        if raw is None:
+            return None
+        value, params = multipart.parse_content_disposition(raw)
```

Comment:
It seems [only two](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) values here are valid for HTTP headers case: `inline` and `attachment`. May be worth to raise a warning about invalid type or turn the `value` in Enum?

Reply:
According to https://tools.ietf.org/html/rfc6266#section-4.1 any token is valid

Reply:
I like `Enum` but how the change affects `multidict` and existing users.
I suspect `parse_content_disposition` is internal API but not 100% sure.

Reply:
@redixin Yes, but RFC defines more general Content-Disposition definition. Mozilla explicitly splits it into HTTP headers and Multpart headers. The second one utilize `token`  rule completely, but here we are in HTTP response land.